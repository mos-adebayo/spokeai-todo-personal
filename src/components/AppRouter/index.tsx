import React, { FC } from 'react';
import {Switch, Route, Redirect } from 'react-router-dom';
import * as Screens from '../../screens';

interface Props {
  Router: any;
}

const AppRouter: FC<Props> = ({ Router }) => (
  <Router>
    <Switch>
      <Route exact path="/" component={Screens.Home} />
      <Route exact path="/todo/:id" component={Screens.TodoItem} />
      <Redirect to="/" />
    </Switch>
  </Router>
);

export default AppRouter;
