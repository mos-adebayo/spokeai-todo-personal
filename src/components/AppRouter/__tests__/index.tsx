import React from "react";
import { BrowserRouter } from "react-router-dom";
import { shallow } from "enzyme";
import AppRouter from "../index";

describe("<AppRouter />", () => {
  it("should match snapshots", () => {
    const wrapper = shallow(<AppRouter Router={BrowserRouter} />);
    expect(wrapper).toMatchSnapshot();
  });
});
