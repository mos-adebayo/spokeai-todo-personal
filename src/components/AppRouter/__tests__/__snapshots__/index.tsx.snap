// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppRouter /> should match snapshots 1`] = `ShallowWrapper {}`;
